### PR TITLE
Better neo4j service

### PR DESCRIPTION
**Pull request by camrossi**

### CODE REVIEW

File: helm/templates/NOTES.txt [hunk 1]
```diff
@@ -1,17 +0,0 @@
-1. Get the application URL by running these commands:
```
Comment (samiib) [inline helm/templates/NOTES.txt:1]: Can you replace these notes with something that matches what you did in the README. This was pretty handy to see how things got deployed and how to access the UI.

File: values_examples/calico-or-kuberouter-NodePort-extIP.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+apicKeyData: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlDWEFJQkFBS0JnUURKV3N1Mi9OOEJDb0ZkWDFDOUh3cW15QkFJcUE3TExoWEVnQjNXcmM3S245aGxKSHNyCklxMkdxWExWRTNQRkY5ZUZQcU4ybzRMbnlHWkRTNDYreEVvTWM3M2YvTDU0eDBDRTZoT1N0cW1mYmFEOE5ybVYKK21XcUFUeXlrU0o0anpVRWp6bVYyRzRyWmRNbXBvek9VNER0OUE4ZWtHLzRVZ3d4NnZ2aXUwNHhaUUlEQVFBQgpBb0dBRnNkVXdIQys4YWswbFZHUkdyUUdTblBiakg4bW5VbXJZTDExb3JHcThUVkFEOEVleGZSNFdRR1dZaEIvClNrNjJMUGEzRzFiSUdseUxaUldKSW4rbU9LckxMZnZtNEJLOW80YkJmTS9iZUNmbEFQajZhVHoxcmllck5hOU8KY2MvTUM5aGxRNDJJNUJYaU8zRlNSK1hqTU5HQmhzSVNnZkc4Z3ErUXkyUFRUa0VDUVFENkd0S0h5dlFHVVN2SgpyeEhyRDRZZS9qK21lZEc0MVhjRCtoTFcxei9Rd21uRXZYWUNpSUNWM0JURFM5Y1lGZlNSamtlZm5BRUg1akZDCnZhc2tndmxwQWtFQXpobk9jQ2dTcnFUMmYxQWxVTHl5WE9RSk9CVW1hUGJJRy9HMXE2WWlGWnE5Y1c0YmRRL08KV2N1RVp2VThTT3B1alpWNm41SktiWDJmanVxZmtkM2NuUUpBQWtOVjNsYXV5SlBaY3BTNzg4V0xNQ0thenRYUgpXUHZOZk9LNVl1cEZzSXpoeWg3d3Yyc2V4cnhjRDd4UzhPdlo3NnM2NkszUHNKT2JwSC8wMlhxNTZRSkFkY2cvCnRzN2Y1bGkxMmdvbVdSNk5YQktoeGlBb0pFSXNGL1FwVzJBRkNJMjNCL1RoMGlOblROL0cyY1A2WUwxQTByOEQKSmVCSGtvMEJPMzZRRko5UzdRSkJBTkRFZ01FRG5KZG43cEJNYzNOeFBuNmpNb0RQaUgrNEFKQjlJRXpBbjZrUwpUbnkyNDFhakFoV2VNMTJlUXhMWXd6Z3FkSm02aFMzRkZzUkRnbUR2cmlFPQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQ=="
```
Comment (samiib) [inline values_examples/calico-or-kuberouter-NodePort-extIP.yaml]: As discussed in the previous messages from 12/07 last year I am just emphasizing on this again that let’s not expose secret keys and passwords even if they’re examples specifically in an opensource project. You can have example config files but there’s absolutely no reason for example key data to be added.

**Comment by samiib** [comment]
How about try using consistent commit messaging. 😊 If you are testing something you don’t always need to commit it with a commit message that only contains “test”. Trying to track down what had changed in the issues was really confusing until I found your unlinked test commits.

File: values_examples/calico-or-kuberouter-NodePort-extIP.yaml (hunk 2 above)
Comment (camrossi) [reply values_examples/calico-or-kuberouter-NodePort-extIP.yaml]: Keys removed. Anyway are just the one to access my APIC that are deleted every time a new cluster is deployed

File: values_examples/calico-or-kuberouter-NodePort-extIP.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    default:
+      externalTrafficPolicy: Local  # Ensure the Service IP is advertised as a /32 where supported (Calico or Kube-Router)
+      type: NodePort
+      externalIPs:
+        - 192.168.14.2
+  neo4j:
+    resources:
+      cpu: "1"
+      memory: "2Gi"
+    password: me-change
```
Comment (samiib) [inline values_examples/calico-or-kuberouter-NodePort-extIP.yaml]: As discussed can we please remove the password from all example files or leave them as empty strings

**Review by samiib — CHANGES_REQUESTED** [review]

File: helm/templates/NOTES.txt (hunk 1 above)
Comment (camrossi) [reply helm/templates/NOTES.txt:1]: Done

**Review by samiib — APPROVED** [review]

**Pull request merged**